app/helpers/kaui/date_helper.rb

```diff
@@ -22,5 +22,24 @@ def format_date(date, timezone)
     def truncate_millis(date_s)
       DateTime.parse(date_s).strftime('%FT%T')
     end
+
+    def current_time(timezone=nil)
+
+      # if no timezone is passed return the time as it
```

Comment:
I believe we should always have access to the `@account` object, see the `before_action` here: https://github.com/killbill/killbill-admin-ui/blob/f3b0a2d82d0a5fb4dcc33a2cd4f42c1d0288d3b1/app/controllers/kaui/engine_controller.rb#L37-L39